src/main/java/com/epam/esm/dao/SqlProvider.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.esm.dao;
+
+public class SqlProvider {
```

Comment:
As far I know, SQL in separate class is a bad practice. http://www.javapractices.com/topic/TopicAction.do?Id=105
For your small application, it's ok to store in DAO classes. https://stackoverflow.com/questions/1661921/java-programming-where-should-sql-statements-be-stored

Reply:
Done, now i store SQL int DAO class